python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -1045,8 +1207,170 @@ router bgp {{ router_bgp.as }}
+{%                 endif %}
+      {{ redistribute_ospf_match }}
+{%             endif %}
+{%             if redistribute_var.ospfv3.enabled is arista.avd.defined(true) %}
+{%                 set redistribute_ospfv3 = "redistribute ospfv3" %}
+{%                 if redistribute_var.ospfv3.include_leaked is arista.avd.defined(true) %}
+{%                     set redistribute_ospfv3 = redistribute_ospfv3 ~ " include leaked" %}
+{%                 endif %}
+{%                 if redistribute_var.ospfv3.route_map is arista.avd.defined %}
+{%                     set redistribute_ospfv3 = redistribute_ospfv3 ~ " route-map " ~ redistribute_var.ospfv3.route_map %}
+{%                 endif %}
+      {{ redistribute_ospfv3 }}
+{%             elif redistribute_var.ospfv3.match_internal.enabled is arista.avd.defined(true) %}
+{%                 set redistribute_ospfv3 = "redistribute ospfv3 match internal" %}
+{%                 if redistribute_var.ospfv3.match_internal.include_leaked is arista.avd.defined(true) %}
+{%                     set redistribute_ospfv3 = redistribute_ospfv3 ~ " include leaked" %}
+{%                 endif %}
+{%                 if redistribute_var.ospfv3.match_internal.route_map is arista.avd.defined %}
+{%                     set redistribute_ospfv3 = redistribute_ospfv3 ~ " route-map " ~ redistribute_var.ospfv3.match_internal.route_map %}
+{%                 endif %}
+      {{ redistribute_ospfv3 }}
+{%             endif %}
+{%             if redistribute_var.ospfv3.match_external.enabled is arista.avd.defined(true) %}
+{%                 set redistribute_ospfv3_match = "redistribute ospfv3 match external" %}
+{%                 if redistribute_var.ospfv3.match_external.include_leaked is arista.avd.defined(true) %}
+{%                     set redistribute_ospfv3_match = redistribute_ospfv3_match ~ " include leaked" %}
+{%                 endif %}
+{%                 if redistribute_var.ospfv3.match_external.route_map is arista.avd.defined %}
+{%                     set redistribute_ospfv3_match = redistribute_ospfv3_match ~ " route-map " ~ redistribute_var.ospfv3.match_external.route_map %}
+{%                 endif %}
+      {{ redistribute_ospfv3_match }}
+{%             endif %}
+{%             if redistribute_var.ospfv3.match_nssa_external.enabled is arista.avd.defined(true) %}
+{%                 set redistribute_ospfv3_match = "redistribute ospfv3 match nssa-external" %}
+{%                 if redistribute_var.ospfv3.match_nssa_external.nssa_type is arista.avd.defined %}
+{%                     set redistribute_ospfv3_match = redistribute_ospfv3_match ~ " " ~ redistribute_var.ospfv3.match_nssa_external.nssa_type %}
+{%                 endif %}
+{%                 if redistribute_var.ospfv3.match_nssa_external.include_leaked is arista.avd.defined(true) %}
+{%                     set redistribute_ospfv3 = redistribute_ospfv3 ~ " include leaked" %}
```

Comment:
This should be `redistribute_ospfv3_match`

Reply:
updated